app/controllers/profile_controller.rb

```diff
@@ -8,4 +8,7 @@ def show
     end
   end
   
+  def register
+    @dorms = [ "Schönherz Zoltán Kollégium", "Nagytétényi úti Kollégium", "Vásárhelyi", "Kármán", "Külsős" ]
```

Comment:
Ezt a listat pedig erdemes kirakni valami konfiguracioba. Lehet siman az application.rb-be is:

```ruby
config.x.dorms = [
  ...
]
```